src/main/scala/tilelink/Broadcast.scala

```diff
@@ -391,45 +559,81 @@ object TLBroadcast
 }
 
 class ProbeTrackInfo(val caches: Int) extends Bundle {
+  /** how many clients left need to be probed. */
   val count   = UInt(log2Ceil(caches+1).W)
+  /** probe which client now. */
   val cacheOH = UInt(caches.W)
 }
 
+/** Tracker to track trans */
```

Comment:
I would say something here about how some people might call this a Miss Status Handling Register (MSHR)

Reply:
Yes, I finally understand it after documenting to MSHR of InclusiveCache, I found they have a similar behavior.